lib/include/hse_ikvdb/kvdb_rparams.h

```diff
@@ -75,10 +75,11 @@ struct kvdb_rparams {
 
     uint32_t dur_bufsz_mb;
     uint32_t dur_intvl_ms;
-    uint8_t  dur_throttle_lo_th;
-    uint8_t  dur_throttle_hi_th;
+    uint32_t dur_sz_bytes;
     bool     dur_enable;
     bool     dur_buf_managed;
+    uint8_t  dur_throttle_lo_th;
```

Comment:
Seems like we could have an 8 bit hole after dur_mclass. Anything else that can be moved around?